frappe/patches/v14_0/remove_db_aggregation.py

```diff
@@ -0,0 +1,25 @@
+import frappe
+import re
+
+
+def execute():
+    _sub_aggregation("frappe.db.max", "frappe.qb.max")
+    _sub_aggregation("frappe.db.min", "frappe.qb.min")
+    _sub_aggregation("frappe.db.sum", "frappe.qb.sum")
+    _sub_aggregation("frappe.db.avg", "frappe.qb.avg")
+
+
+def _sub_aggregation(function, subtitution):
+    scripts = frappe.get_all(
+        "Server Script",
+        filters={"script": ("like", f"%{function}%")},
```

Comment:
I feel we can cut down all this effort by 3/4 if we search for, and replace all functions at once?

Reply:
I reduced the no. of calls e503d81

Reply:
There were issues with that, so I made the changes via https://github.com/frappe/frappe/pull/14457/commits/43fc713dd7ef881f8f76632b266965d94c98de21

Reply:
😢